components/blobserve/cmd/run.go

```diff
@@ -57,6 +57,27 @@ var runCmd = &cobra.Command{
 
 		reg := prometheus.NewRegistry()
 
+		var (
+			clientRequestsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I'm not sure what is the point of tracking total if we already track duration. Each histogram has a counter already 🤔

Reply:
histograms have higher cardinality, so we may need to disable them, counter are less expensive